src/Breadcrumb.js

```diff
@@ -1,27 +1,61 @@
 import classNames from 'classnames';
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import BreadcrumbItem from './BreadcrumbItem';
 import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
 
+const propTypes = {
+  /**
+   * ARIA label for the nav element
+   * https://www.w3.org/TR/wai-aria-practices/#breadcrumb
+   */
+  label: PropTypes.string,
+  /**
+   * Optional prop that hold properties that will be spread to
+   * the list element
+   */
+  listProps: PropTypes.object
+};
+
+const defaultProps = {
+  label: 'breadcrumbs',
+  listProps: {}
+};
+
 class Breadcrumb extends React.Component {
   render() {
-    const { className, ...props } = this.props;
+    const {
+      bsClass,
+      listProps,
+      children,
+      className,
+      label,
+      ...props
+    } = this.props;
     const [bsProps, elementProps] = splitBsProps(props);
 
     const classes = getClassSet(bsProps);
 
     return (
-      <ol
-        {...elementProps}
-        role="navigation"
-        aria-label="breadcrumbs"
+      <nav
+        aria-label={label}
         className={classNames(className, classes)}
-      />
+        {...elementProps}
+      >
+        <ol
+          role="navigation"
+          className={bsClass}
+          children={children}
+          {...listProps}
```

Comment:
I think it's ok for `children` to not be overridden by `listProps` since a user can control them directly already

Reply:
Umm, I needed to add that because the `children` are the `BreadCrumb.Item`s

Reply:
sorry I mean lets be a bit more intentional about how `listProps` is spread in. right now if a user provides it it will override `className` and `children` on `ul` which isn't what we want. I would move `children` down and express it normally e.g. `<ul>{children}</ul` and use `classNames(bsClass, listProps.className)`

Reply:
Yup, agreed. Didn't thought about that thanks for the clarificaiton.